centralserver/i18n/management/commands/update_language_packs.py

```diff
@@ -558,15 +558,22 @@ def produce_outputs(src_po_files, dest_path, lang_code):
                 js_po_file.save_as_mofile(js_mo_file)
             else:
                 # Make sure we only concatenate .po files of the same version that we need.
-                versioned_po_filename = os.path.join("versioned", "%s-django") % (version,)
-                kalite_po_filename = os.path.join("KA Lite UI", "kalite-%s.po") % (lang_code,)
+                versioned_po_filename = os.path.join("kalite-%s", "versioned", "%s-django") % (lang_code, version,)
+                kalite_po_filename = os.path.join("kalite-%s", "KA Lite UI", "kalite-%s.po") % (lang_code, lang_code,)
                 if versioned_po_filename in src_file or kalite_po_filename in src_file:
-                    logging.debug('Concatenating %s with %s...' % (src_file, build_file))
-                    src_po = polib.pofile(src_file)
+                    logging.debug("Concatenating %s with %s..." % (src_file, build_file))
+                    src_po = polib.pofile(
+                        src_file)
                     build_po.merge(src_po)
                 else:
-                    logging.debug("Ignoring %s because it's NOT for version %s." %
-                                  (src_file, version,))
+                    if os.path.basename(src_file):
```

Comment:
This will catch all `.po` files passed to it, including other versions of ka-lite crowdin .po files.

There must be a better way to determine if it's a khan academy .po or not.